app/views/items/show.html.haml

```diff
@@ -12,7 +12,16 @@
         = link_to @item.url, target: :_blank do
           = item_thumbnail(@item, request)
         = link_to @item.url, target: :_blank, class:'ViewObject' do
-          View Object
+          Get full
+          - if @item.type.present? and @item.type.is_a? String
+            = @item.type
+          - else
+            object
```

Comment:
Do we want to change this to `item` instead of `object`? (I might have just missed something about why we decided to keep the wording the same).